### ISSUE TITLE
SF5 : error with dumping command 

### ISSUE THREAD

**Issue by Cryde** [FriendsOfSymfony/FOSJsRoutingBundle#368]
Hi 

I've just update with the new release and I've got this error : 

```
 The service "fos_js_routing.dump_command" has a dependency on a non-existent parameter "kernel.root_dir". Did you mean one of these: "kernel.project_dir", "kernel.cache_dir", "kernel.logs_dir"? 
```

I think this is linked with that : https://symfony.com/blog/new-in-symfony-4-2-important-deprecations  
Thanks for the update anyway

**Comment by tobias-93** [comment]
Sorry for that, fixed with #369

**Comment by Cryde** [comment]
@tobias-93  thanks for the quick response & for this bundle :D !!
